docker/docker-compose.yml

```diff
@@ -38,9 +42,14 @@ services:
             - "traefik.enable=true"
 
 volumes:
-    postgres-events:
-        driver: "local"
+    postgres-oms-events:
```

Comment:
won't this break the current system? :D 
also this: there's `postgres-oms-events` and `oms-events-media`. we can make the volumes named in the same way, like prefixing or suffixing everything with the service name. not sure if it should be done here as well, but given that this already includes volume name change for the working system, why not.

Reply:
ah no, it shouldn't break anything as it's mounted to the host filesystem